trane/core/utils.py

```diff
@@ -164,14 +164,32 @@ def _generate_possible_operations(
         transformation_operations,
         filter_operations,
     ):
+        if (
+            len(
+                {
+                    agg_operation.__name__,
```

Comment:
These could be converted to the actual python objects, rather than relying on strings